packages/standalone/ui/src/styles/global.css

```
@@ -1,31 +1,34 @@
-@custom-variant dark (&:where([data-theme='dark'], [data-theme='dark'] *));
+/* Preflight is deliberately NOT imported: this bundle is CONTENT inside the
+ * Viewer document, which ships its own global reset. Two unscoped resets fight
+ * in load order. The equivalent reset lives under #operator-root below. */
+@import 'tailwindcss/theme.css' layer(theme);
+@import 'tailwindcss/utilities.css' layer(utilities);
 
 /* MAMA Operator theme tokens -- components must consume ONLY these tokens,
  * never hardcoded hex in TSX (learning: mama-dual-color-system).
- * Brand alignment with the legacy palette is a one-token swap on --color-agent. */
+ * Palette is the Viewer's: MAMA yellow on warm off-white, light-only. */
 @theme {
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@theme`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)